infrastructure/static/js/mm-webflow.js

```diff
@@ -844,6 +848,11 @@ function mmWebflow(js) {
 
         $(".detail-button_wrapper").hide();
         $(".subsection-chart__detail").hide();
+
+        var summaryYear = js.summary_year
+        if (summaryYear.split('/')[0] > implementYear.split('/')[0]) {
+            $(".project-details__info-message").parent().append('This project was last updated with ' + implementYear + ' data. Please see the search page for the latest figures.');
```

Comment:
this project _should_ be updated with the latest figures if this project exists in later years, right?

So the message should perhaps refer to "latest projects" rather than "latest figures"

Reply:
nice thinking though!

Reply:
Resolving comments that were addressed to tidy this PR